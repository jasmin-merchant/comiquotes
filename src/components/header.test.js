import React from "react"
import renderer from "react-test-renderer"

import Header from './header.component'

describe("Header", () => {
  it("renders header correctly", () => {
    const tree = renderer
      .create(<Header />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})