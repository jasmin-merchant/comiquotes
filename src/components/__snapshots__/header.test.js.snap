// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="p-4 bg-transparent fixed"
>
  <div
    className="text-xl font-audiowide tracking-wider"
  >
    <div
      className="logo-comi inline-block p-3 pl-6 pr-1 text-white dark:text-stone-900 bg-stone-900 dark:bg-white border-4 border-r-0 border-stone-900 dark:border-white"
    >
      Comi
    </div>
    <div
      className="quotes inline-block p-3 pl-1 pr-6 bg-white dark:bg-red-900 text-red-900 dark:text-white border-4 border-l-0 border-red-900 "
    >
      Quotes
    </div>
  </div>
</header>
`;
